dali/test/python/decoder/test_video.py

```diff
@@ -185,3 +185,35 @@ def test_pipeline():
     for _ in range(iters):
         video_pipeline_0.run()
         video_pipeline_1.run()
+
+
+@params('cpu', 'gpu')
+def test_file_names_meta(device):
```

Comment:
Maybe the name of this test or some comment could mention `SourceInfo`? It took me a minute to figure out what is tested here?

Reply:
Done